### PR TITLE
add chinese version

### PR DESCRIPTION
**Pull request by soulsoul**
#1290

### CODE REVIEW

**Comment by moonming** [comment]
Please check CI

**Comment by membphis** [comment]
we need to update the `REAME_CN.md` too

https://github.com/apache/incubator-apisix/blob/master/README_CN.md#%E5%BF%AB%E9%80%9F%E4%B8%8A%E6%89%8B

File: doc/getting-started-cn.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+[English](getting-started.md)
+# 开速入门指南
```
Comment (membphis) [inline doc/getting-started-cn.md]: `快速` ??

File: doc/getting-started-cn.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+本指南的目的是介绍如何使用 APISIX 来配置出一个安全的可以对外提供服务的 API 。当您读完本指南 ，你需要自己安装一下 APISIX 应用， 并准备好一个可以对外提供服务的 API ，该服务将由 API key 进行访问保护。
```
Comment (membphis) [inline doc/getting-started-cn.md]: `当您读完本指南 ，` -> `当您读完本指南，`

File: doc/getting-started-cn.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+
+以下 GET 请求可以用来指南的教学 。 该服务可以回显发送到这个 API 的传参 。
```
Comment (membphis) [inline doc/getting-started-cn.md]: `南的教学 。` => `南的教学。`

File: doc/getting-started-cn.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+- Query Parameters: foo1, foo2
+
+## 前提
+
+- 本指南使用 docker 和 docker compose 来安装 APISIX 。 但是 ， 如果您已经以其他方式安装了 APISIX ，您只需跳到 [第二步](getting-started-cn.md#第二步:-在-APISIX-中设置路由) 。
+- Curl：指南使用 Curl 命令进行 API 测试，但是您也可以使用您选择的任何其他工具（ 例如 Postman ）。
+
+## 第一步: 安装 APISIX
+
+APISIX 可以多种操作环境中安装 。 [以下文档](how-to-build.md#installation-via-source-release) 显示了多个平台中的安装步骤 。
```
Comment (membphis) [inline doc/getting-started-cn.md]: `一下文档` -> `如何安装文档`

`how-to-build.md` -> `how-to-build-cn.md`

File: doc/getting-started-cn.md (hunk 3 above)
Comment (membphis) [reply doc/getting-started-cn.md]: please fix some other similar point

File: doc/getting-started-cn.md (hunk 2 above)
Comment (soulsoul) [reply doc/getting-started-cn.md]: No space for symbols and words in Chinese？

File: doc/getting-started-cn.md (hunk 4 above)
Comment (soulsoul) [reply doc/getting-started-cn.md]: got it

File: doc/getting-started-cn.md [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+
+    ```bash
+    curl -i -X GET "http://{apisix_server.com}:{port}/services/users/getAllUsers?limit=10" -H "Host: example.com"
+    ```
+
+- Upstream 信息：
+
+    Upstream 是一个虚拟主机抽象，它根据配置规则在给定的一组服务节点上执行负载平衡。
+    因此，单个上游配置可以由提供相同服务的多个服务器组成。每个节点将包括一个 key（地址/ip:port）和一个 value （节点的权重）。
+    服务可以通过循环或一致哈希（cHash）机制进行负载平衡。
```
Comment (membphis) [inline doc/getting-started-cn.md]: `循环` -> `轮训`

File: doc/getting-started-cn.md [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+    配置路由时，可以直接设置 Upstream 信息，也可以使用服务抽象来引用 Upstream 信息。
+
+- 各种插件
+
+    插件允许您扩展 APISIX 的功能，并实现可以与 HTTP request / response 生命周期接口的任意逻辑。
+    因此，如果您想对 API 进行身份验证，那么您可以使用密钥验证插件来对每个请求强制进行身份验证。
+
+### 设置 Upstream
+
+执行以下命令在 APISIX 中创建id为 50 的上游信息，并使用 round-robin 机制进行负载平衡。
```
Comment (membphis) [inline doc/getting-started-cn.md]: `中创建id为`

missing space between Chinese and Egnlish

File: doc/getting-started-cn.md [hunk 7]
```diff
@@ -0,0 +1,256 @@
...
+    "username": "john",
+    "plugins": {
+        "key-auth": {
+            "key": "superSecretAPIKey"
+        }
+    }
+}'
+```
+
+现在，让我们将服务配置为包含密钥验证插件。
```
Comment (membphis) [inline doc/getting-started-cn.md]: `密钥验证插件`  => `KEY 验证插件`

Comment (membphis) [reply doc/getting-started-cn.md]: please fix other similar point

File: doc/getting-started-cn.md [hunk 8]
```diff
@@ -0,0 +1,256 @@
...
+由于 route 由密钥验证插件保护，前一个访问 API 的 curl 命令将产生未经授权的访问错误。
+现在使用下面的命令安全地访问请求。
+
+```bash
+curl -i -X GET http://127.0.0.1:9080/get -H "Host: httpbin.org" -H 'apikey: superSecretAPIKey'
+```
+
+## 为 route 添加前缀
+
+
```
Comment (membphis) [inline doc/getting-started-cn.md]: one blank line is enough

File: doc/getting-started-cn.md [hunk 9]
```diff
@@ -0,0 +1,256 @@
...
+}'
+```
+
+现在可以使用以下命令调用路由：
+
+```bash
+curl -i -X GET http://127.0.0.1:9080/samplePrefix/get?param1=foo&param2=bar -H 'apikey: superSecretAPIKey'
+```
+
+## APISIX 控制台
```
Comment (membphis) [inline doc/getting-started-cn.md]: please use `Apache APISIX`

File: doc/getting-started-cn.md (hunk 5 above)
Comment (soulsoul) [reply doc/getting-started-cn.md]: 轮询

**Comment by membphis** [comment]
many thanks,  merged already

**Pull request merged**